src/request/create/index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="Content-Security-Policy"
+          content="default-src 'self' 'unsafe-eval' https://cdn.nimiq.com;
```

Comment:
does that mean that `unsafe-eval` is allowed?

Reply:
There was something about it being required for WebAssembly to work under iOS, IIRC.

Reply:
More or less. Core needs it to determine if it has to use it's babel-built fallback version.